### DISCUSSION TITLE
facing t.useId is not a function error

### DISCUSSION THREAD

**Discussion by shiraazAh** [pacocoursey/cmdk#43] [Q&A] [answered]
Just wanted to confirm if this is a known bug or not, whenever I'm trying to use the package its giving me the error `t.useId is not a function` as shown in the image below.  
![t useId is not a function - Google Search - Google Chrome 29-08-2022 23_17_35](https://user-images.githubusercontent.com/35837216/187266628-b1355e80-88b2-42a9-a99f-ff457bb8e868.png)

Im using react & react-dom version 17

**Comment by pacocoursey** [comment, marked as answer]
This library requires React 18, as stated in the [FAQ](https://github.com/pacocoursey/cmdk#faq) and the [`peerDependencies`](https://github.com/pacocoursey/cmdk/blob/3760c39c2081168e2cc05b09ed568f954a60a289/cmdk/package.json#L17-L20).
